README.md

```diff
@@ -21,7 +21,7 @@ This project is organized by our [Governance structure](docs/GOVERNANCE.md).
 ### Participating and contributing
 
 We welcome all to participate in the Beman project through 
-[Discourse](https://discourse.boost.org/t/welcome-to-beman-project-development/3) discussions or GitHub contributions. Our discourse community is the best way to stay up to
+[Discourse](https://discourse.bemanproject.org/latest) discussions or GitHub contributions. Our discourse community is the best way to stay up to
```

Comment:
Please use one sentence per line, as it would be easier to track when doing changes. I plan to add some linter rule for this, not ready yet.

Reply:
I'm not convinced that "one sentence per line" is the best practice for writing markdown files. Flowing the text at, e.g., 80 characters seems much more common.